content/en/platform/corda/5.0-beta/operating/manual-registration.md

```diff
@@ -0,0 +1,24 @@
+---
+date: '2023-03-15'
+title: "Manual Registration Approval"
```

Comment:
General comment on this and other files; should the filename when created match the title?

Reply:
Yes. I was chopping and changing titles and should have come back to this. Thanks.